### DISCUSSION TITLE
docs/v1.11.x/guide/qos/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2557] [Announcements]
# docs/v1.11.x/guide/qos/

https://kubeovn.github.io/docs/v1.11.x/guide/qos/

**Comment by xinsir88** [comment]
可以针对namespace做流量限速吗？

**Reply by oilbeater** [reply]
namspace 下的 pod 会分布在不同机器，没法做统一限速

**Comment by kingsley8888888888** [comment]
怎么去通过kubectl ko工具去查看QoS的规则

**Reply by oilbeater** [reply]
目前不支持，需要看 pod 的 annotation
